File: python/query.py
import os
from py2neo import neo4j, cypher
graph_db = neo4j.GraphDatabaseService('http://localhost:7474/db/data')
query = 'MATCH a-[:SELLS_TO]->b return a.name,b.name'
data, metadata = cypher.execute(graph_db, query)
for pair in data:
    print("%s sells to %s" % (pair[0], pair[1]))

File: javascript/query.js
var neo4j = require('neo4j');
var db = new neo4j.GraphDatabase('http://localhost:7474');

db.query("MATCH a-[:SELLS_TO]->b return a.name as seller,b.name as buyer", {}, function (err, results) {
  if (err) throw err;
  results.forEach(function (item) {
    console.log(item.seller + ' sells to ' + item.buyer);
  })
});
